services/productService.js

```diff
@@ -0,0 +1,28 @@
+const productDao = require('../models/productDao');
+const userDao = require('../models/userDao');
+
+const createBooking = async (
+  productId,
+  guestId,
+  guestNumber,
+  checkIn,
+  checkOut,
+  totalPrice
+) => {
+  try {
+    const user = await userDao.getUserByKakaoId(guestId);
```

Comment:
Dao단에서 userId를 바로 받아올 수 있도록 수정하면 좋을 것 같네요!
Dao에서 result.id를 반환하면 서비스 단에서 userId 변수로 바로 할당할 수 있을 것 같습니다!
 ```suggestion
    const userId = await userDao.getUserByKakaoId(guestId);
```